src/makeCalculator.js

```diff
@@ -37,7 +37,49 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let output = 0;
+
+  return {
+    add(x) {
+      output += x;
+
+      return this;
+    },
+
+    subtract(x) {
+      output -= x;
+
+      return this;
+    },
+
+    multiply(x) {
+      output *= x;
```

Comment:
Fix it everywhere
```suggestion
      this.result *= x;
```

Reply:
`function makeCalculator() {
  let output = 0;

  return {
    result: 0,

    add(x) {
      this.result += x;

      return this;
    },

    subtract(x) {
      this.result -= x;

      return this;
    },

    multiply(x) {
      this.result *= x;

      return this;
    },

    divide(x) {
      if (x !== 0) {
        this.result /= x;
      } else {
        return 'Division by zero!';
      }

      return this;
    },

    reset() {
      this.result = 0;

      return this;
    },

    operate(operation, operand) {
      operation(operand);

      return this;
    },

    get result() {
      return this.result;
    },
  };
}`
I did everything as you said, but jest gives me an error
```
sum > 'operate' should correctly add a single value
-----
TypeError: Cannot read properties of undefined (reading 'result')
```
I tried giving different names to the variables but it didn't help